### DISCUSSION TITLE
Line items reorder

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14815] [How to] [answered]
On my 6.1 there is ability to move line items during advanced module record editing. This is buggy and very uncomfortable for users.

I has checked gitdev and found what there is no ability to move line items. This is final fix for this bug???

**Comment by rskrzypczak** [comment, marked as answer]
This is not a bug, and it works like that on gitdev as well.
The ability to manipulate the order of entries is available after adding the Item number field to the block.
![image](https://user-images.githubusercontent.com/10221999/121492736-6a175e00-c9d7-11eb-93f1-00cbc4f6937d.png)

**Reply by vovpff** [reply]
Ok. What about buggy reorder in 6.1? How to cure it?

**Reply by vovpff** [reply]
@rskrzypczak @mariuszkrzaczkowski Pls check this movie. This is on gitdev. `Item number` added to quotes for ability to reorder lines.

![reorder](https://user-images.githubusercontent.com/8545791/121651496-5d603c00-caa3-11eb-8dee-eee27ce815d9.gif)

**Reply by rskrzypczak** [reply]
Fix: https://github.com/YetiForceCompany/YetiForceCRM/commit/b1baa2b4f9fbad702afd95a6433ddc594833b16b

**Reply by vovpff** [reply]
I will check this fix tomorrow !

**Reply by vovpff** [reply]
I has checked this fix. This works better but sometimes line comment window appears on wrong place 

![reorder_1](https://user-images.githubusercontent.com/8545791/122180648-48175300-ce91-11eb-835e-aa3b4202cd5a.gif)
